Section 2/2.19_Working_With_Numbers.rb

```diff
@@ -0,0 +1,55 @@
+puts 1 + 2
+# 3
+# => nil
+
+# IRB
+# Division between Integers only get back Integer
+puts 10 / 2
+# => 5
+puts 10 / 4
```

Comment:
Why? Shouldn't this be 2.5?

Reply:
10 / 4 is a mathematical operation between two integers, the result is also in the Integer type. We will get 2.5 if we count by ourselves but in the program, the decimal places are auto dropped because Integer is not allowed to have the decimal places.

Reply:
### Why computer counts differently than us?
We use the decimal system because we have ten fingers to count, while machines do not. As a result, we've developed alternative number systems to do the same jobs for the computers which made the computer display information differently. 

For example, Computers support the following number systems:
1. Binary
2. Octal
3. Decimal
4. Hexadecimal

### Why don't we just use Float arithmetic for all calculations?
Integer arithmetic is faster. And for a given range integers consume less memory because integers don't need to represent non-integer values.